pretyping/detyping.ml

```diff
@@ -447,7 +467,7 @@ let detype_case computable detype detype_eqns testdep avoid data p c bl =
   let synth_type = synthetize_type () in
   let tomatch = detype c in
   let alias, aliastyp, pred=
-    if (not !Flags.raw_print) && synth_type && computable && not (Int.equal (Array.length bl) 0)
+    if (not (Printoptions.printing_all ())) && synth_type && computable && not (Int.equal (Array.length bl) 0)
```

Comment:
What's going on here?

Reply:
I should just be using `printing_synth` here, the ref flag should not be used.